cmd/rook/ctl/multus/validation/validation.go

```diff
@@ -0,0 +1,213 @@
+			"If you aren't sure what to choose for this value, add 1 for each additional OSD beyond 3.")
+
+	// flags for 'validation cleanup'
+	// none
+}
+
+func runValidation(ctx context.Context) {
+	results, err := validationConfig.Run(ctx)
+	report := results.SuggestedDebuggingReport()
+
+	// success/failure message
+	fmt.Print("\n")
+	switch {
+	case err != nil:
+		fmt.Printf("RESULT: multus validation test failed: %v\n\n", err)
+	case report == "":
+		fmt.Print("RESULT: multus validation test succeeded!\n\n")
+		runCleanup(ctx)
+		os.Exit(0) // success!
+	case report != "":
+		// suggestions are bad
+		fmt.Print("RESULT: multus validation test succeeded, but there are suggestions\n\n")
+	}
+
+	// output report suggestions
+	fmt.Print(report + "\n")
+
+	// help users help us help them
+	fmt.Println("leaving multus validation test resources running for manual debugging")
+	fmt.Print(`
+for assistance debugging, collect the following into an archive file:
+  - Output of this utility
+  - Network Attachment Definitions (NADs) used by this test
+  - A write-up describing the network configuration you are trying to achieve including the
+      intended network for Ceph public/client traffic, intended network for Ceph cluster traffic,
+      interface names and CIDRs for both networks, and any other details that are relevant.
+  - 'ifconfig' output from at least one Kubernetes worker node
+  - 'kubectl get pods -o wide' output from the test namespace
+  - 'kubectl describe pods' output from the test namespace
+  - 'kubectl get pods -o yaml' output from the test namespace
+  - 'kubectl get daemonsets' output from the test namespace
+  - 'kubectl describe daemonsets' output from the test namespace
+  - 'kubectl get daemonsets -o yaml' output from the test namespace
+  - 'kubectl logs multus-validation-test-web-server' output from the test namespace
+  - 'kubectl get nodes -o wide' output
+`)
+
+	os.Exit(1)
+}
+
+func runCleanup(ctx context.Context) {
+	fmt.Printf("cleaning up multus validation test resources in namespace %q\n", validationConfig.Namespace)
+	results, err := validationConfig.CleanUp(ctx)
+	if err != nil {
+		fmt.Printf("multus validation test cleanup failed: %v\n\n", err)
+		fmt.Println(results.SuggestedDebuggingReport())
+		return
+	}
+	fmt.Print("multus validation test resources were successfully cleaned up\n")
+}
+
+// custom flag types
+
+// implements pflag.Value interface to validate and set resource timeout and enforce nonzero
+type timeoutMinutes time.Duration
+
+func (t *timeoutMinutes) String() string { return time.Duration(*t).String() }
+func (t *timeoutMinutes) Set(v string) error {
+	i, err := strconv.Atoi(v)
+	if err != nil {
+		return err
+	}
+	if i < 1 {
+		return fmt.Errorf("timeout must be greater than 0")
+	}
+	*t = timeoutMinutes(time.Duration(i) * time.Minute)
+	return nil
+}
+func (t timeoutMinutes) Type() string {
+	return "timeoutMinutes"
```

Comment:
```suggestion
	return "timeout Minutes"
```

Reply:
It's a type, which can't have spaces in Go